CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [[2.0.1](https://github.com/multiversx/mx-lite-wallet-dapp/pull/90)] - 2025-05-15
```

Comment:
This is the wrong date, which should be today, and the wrong reference as well, since the current `Pull Request` should be referenced instead.